### DISCUSSION TITLE
nodrag not working on Custom Edge?

### DISCUSSION THREAD

**Discussion by singhshemona** [xyflow/xyflow#3259] [Q&A] [answered]
Hey everyone, I noticed a bug with my custom edge component: I am unable to highlight text because the canvas drags along with it. I put a `nodrag` in the `className` property of the element, but that's not working. What's strange is that my custom node is setup almost exactly the same but is working correctly. Anyone know what might be wrong?

App: https://network-pi.vercel.app/

Code for custom edge: https://github.com/singhshemona/network/blob/main/src/components/TextUpdaterEdge/TextUpdaterEdge.tsx

Steps to reproduce bug:
1. Click on custom edge to activate text box view
2. Try to highlight text in the text box, but you cannot

**Comment by bcakmakoglu** [comment, marked as answer]
I think you're looking for `nopan` instead of `nodrag`

**Reply by singhshemona** [reply]
Yes! That was it, thank you 🙏🏽 🎉
